server/main_chatbot.py

```
@@ -378,6 +378,11 @@ def save_incident_thought(content: str, force: bool = False):
                                         "VALUES (%s, %s, %s, %s)",
                                         (incident_id, datetime.now(), cleaned_text, "analysis")
                                     )
+                                    set_rls_context(cursor, conn, user_id, log_prefix="[BackgroundChat]")
+                                    cursor.execute(
+                                        "UPDATE incidents SET updated_at = %s WHERE id = %s",
+                                        (datetime.now(), incident_id),
```

Ruff (0.15.12)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)